src/Lykke.Tools.AssetMigrator/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Threading.Tasks;
+using Lykke.Logs;
+using Lykke.Logs.Loggers.LykkeConsole;
+using Lykke.Tools.AssetMigrator.Implementations;
+using Microsoft.Extensions.DependencyInjection;
+
+
+namespace Lykke.Tools.AssetMigrator
+{
+    internal static class Program
+    {
+        
+        private static async Task<int> Main(string[] args)
+        {
+            int resultCode;
+            
+            try
+            {
+                var services = ConfigureServices();
+                var serviceProvider = services.BuildServiceProvider();
+                var rootCommand = serviceProvider.GetService<IRootCommand>();
+                var app = rootCommand.Configure();
+
+                resultCode = app.Execute(args);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+
+                resultCode = 1;
+            }
+
+            // Flushing logger
+            await Task.Delay(100);
```

Comment:
dispose service provider in the end. then you don't need to flush logger with Task.Delay